mrp_bom_reference_selection/views/mrp_bom_view.xml

```diff
@@ -2,7 +2,7 @@
 <openerp>
     <data>
 
-        <record id="mrp_bom_tree_view" model="ir.ui.view">
+        <record id="mrp.mrp_bom_tree_parent_view" model="ir.ui.view">
```

Comment:
By doing this we create a base view so that both modules can be installed separatelly and they won't collide between each other.

Correct option will be to have the view in `mrp` module but, as it is not, I think this is the best option.